source/components/localisation/useLocalisation.ts

```diff
@@ -10,11 +11,12 @@ const API = '/geolocation'
 
 export default () => {
 	const dispatch = useDispatch()
-
-	const localisation = useSelector((state) => state.localisation)
+	const localisation = useSelector((state: AppState) => state.localisation)
 
 	useEffect(() => {
-		if (localisation?.country != null) return
+		if (localisation?.country != null) {
```

Comment:
C'est une bonne pratique de privilégier `===` et `!==`
```suggestion
		if (localisation?.country !== null) {
```

Reply:
Hmm j'utilise la comparaison non-strict pour gérer le cas où `country` est `undefined`.